### PR TITLE
Drop Python2.7 support.

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

**Review by kencochrane — COMMENTED** [review]
LGTM assuming it passes all the tests.

File: setup.py [hunk 1]
```diff
@@ -58,7 +59,6 @@ def get_package_data(package):
     package_data=get_package_data("defender"),
     install_requires=["Django>=1.8,<3.2", "redis<3.6"],
```
Comment (smithdc1) [inline setup.py:60]: Add `python_requires`? I've had comments from users before about releasing versions which drop old Python versions without this. If they don't have versions pinned they get upgraded to a (known) failing version.

**Comment by smithdc1** [comment]
I was just going to comment on `super()` :+1: 

There's also the following: 
- A couple of files (setup.py, and a couple of the migration files) with the utf coding comment at the top. 
- a `__future__` import in `cleanup_django_defener.py` which can be removed

File: setup.py (hunk 1 above)
Comment (hramezani) [reply setup.py:60]: Good point :+1: . I added `python_requires`

**Review by smithdc1 — APPROVED** [review]

**Pull request merged**